tests/installation/partitioning_firstdisk.pm

```diff
@@ -34,8 +45,16 @@ sub take_first_disk_storage_ng {
     assert_screen 'partition-scheme';
     send_key $cmd{next};
     # select btrfs file system
-    assert_and_click 'default-root-filesystem';
-    assert_and_click "filesystem-btrfs";
+    if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
```

Comment:
Like above, I don’t see why ssh-x can’t use an x dependant function

Reply:
Yep, my mistake. It should be for VIDEOMODE=text only. For yast installation with textmode, assert_and_click does not work.